plugins/wacom-usb/fu-wac-device.c

```diff
@@ -583,7 +583,7 @@ fu_wac_device_write_firmware(FuDevice *device,
 		}
 
 		/* calculate expected checksum and save to device RAM */
-		csum_local[i] = fu_wac_calculate_checksum32le_bytes(blob_block);
+		csum_local[i] = fu_common_sum32w_bytes(blob_block, G_LITTLE_ENDIAN);
```

Comment:
`fu_wac_calculate_checksum32le_bytes()` returns guint32 as G_LITTLE_ENDIAN while new API is a host-based.

Reply:
Yes, agreed!

    - 		csum_local[i] = fu_common_sum32w_bytes(blob_block, G_LITTLE_ENDIAN); 
    +		csum_local[i] = GUINT32_TO_LE(fu_common_sum32w_bytes(blob_block, G_LITTLE_ENDIAN));

right?